ui/client/src/components/Header.tsx

```diff
@@ -14,24 +14,24 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import { AppBar, Box, Button, Grid2, IconButton, Tab, Tabs, ToggleButton, ToggleButtonGroup, Toolbar, Tooltip, useMediaQuery, useTheme } from "@mui/material";
+import { AppBar, Box, Button, Grid2, IconButton, Tab, Tabs, Toolbar, useMediaQuery, useTheme } from "@mui/material";
 import { useContext, useState } from "react";
 import { useTranslation } from "react-i18next";
 import { useLocation, useNavigate } from "react-router-dom";
-import Brightness4Icon from '@mui/icons-material/Brightness4';
 import { ApplicationContext } from "../contexts/ApplicationContext";
-import PlayArrowIcon from '@mui/icons-material/PlayArrow';
-import PauseIcon from '@mui/icons-material/Pause';
 import RefreshIcon from '@mui/icons-material/Refresh';
+import { SettingsMenu } from "../menus/Settings";
+import MenuIcon from '@mui/icons-material/Menu';
 
 export const Header: React.FC = () => {
 
-  const { colorMode, autoRefreshEnabled, setAutoRefreshEnabled, refreshRequired, refresh } = useContext(ApplicationContext);
+  const { refreshRequired, refresh } = useContext(ApplicationContext);
   const { t } = useTranslation();
   const navigate = useNavigate();
   const pathname = useLocation().pathname.toLowerCase();
   const theme = useTheme();
   const lessThanMedium = useMediaQuery(theme.breakpoints.down("md"));
+  const [anchorEl, setAnchorEl] = useState<null | HTMLElement>(null);
 
   const getTabFromPath = (path: string) => {
     if (path.startsWith('/ui/indexer')) {
```

Comment:
Might be a good idea to put the routes in a map to prevent typos, and replace the strings in places such as https://github.com/LF-Decentralized-Trust-labs/paladin/blob/main/ui/client/src/App.tsx#L97-L99.

```typescript
export const AppRoutes = {
  Indexer: '/ui/indexer',
  Submissions: 'ui/submissions',
  Submission: (submissionId: string) => `ui/submissions/${submissionId}`,
  ...
}
```

Reply:
Great idea, will move those to constants defined in a separate file.